stuffer/s2n_stuffer_pem.c

```diff
@@ -79,16 +79,18 @@ static int s2n_stuffer_pem_read_contents(struct s2n_stuffer *pem, struct s2n_stu
     struct s2n_stuffer base64_stuffer = {0};
     GUARD(s2n_stuffer_init(&base64_stuffer, &base64__blob));
 
+    PRECONDITION_POSIX(s2n_stuffer_is_valid(pem));
     while (1) {
-        char c;
+        /* We need a byte... */
+        S2N_ERROR_IF(s2n_stuffer_data_available(pem) < 1, S2N_ERR_STUFFER_OUT_OF_DATA);
```

Comment:
ENSURE, as I think we want to deprecate S2N_EROR_IF

Reply:
Made it `ENSURE_POSIX`